packages/babel-config/src/__tests__/common.test.ts

```diff
@@ -210,6 +210,41 @@ describe('common', () => {
           `"src/ui"`
         )
       })
+
+      it('gets and formats paths for vite', () => {
+        const apiTSConfig =
+          '{"compilerOptions":{"baseUrl":"./","paths":{"@services/*":["./src/services/*"]}}}'
+        const webTSConfig =
+          '{"compilerOptions":{"baseUrl":"./","paths":{"@ui/*":["./src/ui/*"]}}}'
+
+        vol.fromNestedJSON(
+          {
+            'redwood.toml': '',
+            api: {
+              'tsconfig.json': apiTSConfig,
+            },
+            web: {
+              'tsconfig.json': webTSConfig,
+            },
+          },
+          redwoodProjectPath
+        )
+
+        const typeScriptConfig = parseTypeScriptConfigFiles()
+
+        const apiPaths = getPathsFromTypeScriptConfig(typeScriptConfig.api)
+        expect(ensurePosixPath(apiPaths['@services'])).toMatchInlineSnapshot(
+          `"src/services"`
+        )
+
+        const webPaths = getPathsFromTypeScriptConfig(
+          typeScriptConfig.web,
+          true
+        )
+        expect(ensurePosixPath(webPaths['@ui'])).toMatchInlineSnapshot(
+          `"./src/ui"`
```

Comment:
Hey @esteban-url just to check - why is the `./` required for vite? src/ui should still be working I believe?